acts.spec

```
@@ -0,0 +1,77 @@
+### RPM external acts 44.0.1-cms2
+## INCLUDE cpp-standard
+## INCLUDE microarch_flags
+## INCLUDE compilation_flags
+## INCLUDE compilation_flags_lto
+## INCLUDE cuda-flags
+## INCLUDE rocm-flags
+
+Source: https://github.com/fwyzard/%{n}/archive/refs/tags/v%{realversion}.tar.gz
+Source99: scram-tools.file/tools/eigen/env
+
+BuildRequires: cmake
+Requires: python3
+Requires: xerces-c
+Requires: dd4hep
+Requires: json
+Requires: expat
+Requires: zlib
+Requires: geant4
+Requires: boost
+Requires: eigen
+Requires: root
+%{!?without_cuda:Requires: cuda}
+%{!?without_rocm:Requires: rocm}
+
+%prep
+%setup -n %{n}-%{realversion}
+
+%build
+rm -rf ../build
+mkdir ../build
+cd ../build
+source %{_sourcedir}/env
+
+%define cuda_enabled %{?without_cuda:OFF}%{!?without_cuda:ON}
+%define rocm_enabled %{?without_rocm:OFF}%{!?without_rocm:ON}
+
+cmake ../%{n}-%{realversion} \
+  -DCMAKE_CXX_FLAGS="-fPIC $CMS_EIGEN_CXX_FLAGS %{arch_build_flags} %{selected_microarch} %{lto_build_flags}" \
+  -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} \
+  -DCMAKE_INSTALL_PREFIX="%{i}" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_PREFIX_PATH="${PYTHON3_ROOT};${XERCES_C_ROOT};${DD4HEP_ROOT};${JSON_ROOT};${EXPAT_ROOT};${ZLIB_ROOT};${GEANT4_ROOT};${BOOST_ROOT};${EIGEN_ROOT};${ROOT_ROOT}%{!?without_cuda:;${CUDA_ROOT}}%{!?without_rocm:;${ROCM_ROOT}}" \
```

```suggestion
  -DCMAKE_PREFIX_PATH="%{cmake_prefix_path}" \
```

`cmake_prefix_path` macro expands to the install path of all the dependencies

---

oh, thanks !